app/views/bookings/new.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= simple_form_for() do |f|%>
```

Comment:
simple_form_for()  --> need to pass the parameter. in this case maybe should be @booking

Reply:
The form is done.